src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -77,7 +77,8 @@
             },
             "recreateBuildPlans": {
                 "title": "Recreate Build Plans",
-                "warning": "Build plans will be recreated. All previous changes will be overwritten!",
+                "buttonText": "Build Plan",
```

Comment:
Why is it now only called "Build Plan"? I think it's not clear what functionality hides behind this button

Reply:
The jhiDeleteButton has an action type input that appends information about the performed action. Therefore, the actual button label that is shown to the user is ‘Build Plan Reset’/‘Build-Pläne Zurücksetzen’.